packages/ember-metal/lib/meta.js

```diff
@@ -46,6 +47,24 @@ let members = {
   tag: ownCustomObject
 };
 
+// This is less than ideal, but `runInDebug` is
+// not populated by the time we need it here. So
+// we create a function that has the same signature
+// as the real `runInDebug` so that it is stripped
+// from the production build. Unfornately, this
+// function will remain in production build.
+let runInDebug = (fn) => fn();
```

Comment:
The function never calls the pass function. There is some race when `setDebugFunction` is [called](https://github.com/emberjs/ember.js/blob/28cedad1418bedd2dad44eb56668fa67b9769d86/packages/ember-debug/lib/index.js#L148) and when this module is required.